Tests/SnapshotTestingTests/SnapshotTestingTests.swift

```diff
@@ -1,12 +1,24 @@
 @testable import SnapshotTesting
 import XCTest
 
-#if os(iOS)
+#if os(macOS) || os(iOS)
+import SceneKit
+import SpriteKit
 import WebKit
+
+#if os(iOS)
+typealias View = UIView
 let platform = "ios"
 #elseif os(macOS)
-import WebKit
+typealias View = NSView
```

Comment:
I think this can be dropped since you have it in `SnapshotTesting` and you are doing a testable import?

Reply:
Ah yup, those used to be private!